frontend/ui/otamanager.lua

```diff
@@ -227,6 +229,10 @@ function OTAManager:fetchAndProcessUpdate()
         UIManager:show(InfoMessage:new{
             text = _("KOReader is up to date."),
         })
+    elseif ota_version == -1 then
+        UIManager:show(InfoMessage:new{
+            text = T(_("Device no longer supported.\n\nPlease check %1."), "https://github.com/koreader/koreader/wiki/deprecated-devices")
```

Comment:
Always bothering to leave a period `.` at the end of the URL, people typing it with the period will get a 404.